packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/Page/index.js

```diff
@@ -35,7 +36,12 @@ class PageContainer extends React.Component {
   }
 
   render() {
-    return <Wrapper ref='page' children={this.props.children} />
+    return (
+      /* eslint-disable-next-line react/no-string-refs */
```

Comment:
I added this disabled because I am not sure how the `ref="page"` is been used in this component